### ISSUE TITLE
Peer Review by zs366

### ISSUE THREAD

**Issue by zs366** [vae4/WineAnalysis#5]
Summary of the midterm project:
This project attempts to predict the quality of wine based on features (such as pH, density, residual sugar, volatile acidity, and alcohol content) to better price and supply wine to customers. The group uses a dataset consisting of 6451 different wines and 13 features, as well as the quality ratings and other characteristics of those wines. The group aims to assess which combination of these features lead to different qualities of wine.

Three things I like:
1. It is great that the group reduces the bias of data with bootstrapping to ensure the result is not severely affected by the extreme wine ratings.
2. I like that the group tries out different models (such as least squares, linear regression, and quadratic regression) to find the one they think fits the data best.
3. I like that the group shuffles the dataset before allocating it into a training set and a testing set to ensure the two sets are completely random.

Three areas for improvement:
1. While the dataset seems to be convincing and reliable, it would be great if you also provide the source or the link to the dataset.
2. I see the project has a problem with overfitting. I would love to know more about your detailed plan for avoiding overfitting and underfitting.
3. Although the group provides a detailed preliminary wine analysis, it would be great if you also talk about what the further steps are for this project. Will you try out more models to fit the data?

Overall, great job and I'm curious to see what this group comes up with!
